tests/rule_based_profiler/parameter_builder/test_numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -240,7 +240,7 @@ def test_oneshot_numeric_metric_range_multi_batch_parameter_builder_bobby(
 
     actual_value_01_lower: float = actual_values_01[0]
     actual_value_01_upper: float = actual_values_01[1]
-    expected_value_01_lower: float = -51.7
+    expected_value_01_lower: float = -52.0
```

Comment:
@NathanFarmer Is it of any concern that the value has to change?..

Reply:
This is still WIP. You won't see this in the version that is ready.